private/sass/sites/_content.scss

```diff
@@ -5,6 +5,12 @@ $section-gray: #3c3740;
 $section-cyan: #3aa9a5;
 $section-violet: #61285b;
 
+// pagespeed optimisations
+body {
+    visibility: visible !important;
+    transition: none;
+}
```

Comment:
here you reset back the body, but you do not reset back all the other styles for mq
```diff
+            img { width: 100%; }
+            .sr-only, .skip-links, .navbar-collapse { display: none; }
```